gcs/gcs-generic.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: tekton.dev/v1beta1
+kind: Task
+metadata:
+ name: gcs-generic
+spec:
+  workspaces:
+  - name: credentials
+    description: A secret with a service account key to use as GOOGLE_APPLICATION_CREDENTIALS.
+  params:
+  - name: command
```

Comment:
We can probably combine command and subcommand into a single args param. This might make it easier if there are any actual subcommands (e.g. `acl get` vs command args).

Reply:
(Don't have strong opinions on this though...so open to changing if you feel so!)

Reply:
Keep the two params approach but update the names as "command" and "options"